### DISCUSSION TITLE
In the "explore" tab on mobile when someone stars a repository it says "Started a discussion" instead.

### DISCUSSION THREAD

**Discussion by gibbiemonster** [community/community#19792] [Mobile] [answered]
When you tap the explore tab, you see "Started a discussion" instead of "Starred a repository".

![Screenshot_20220630-212022.png](https://user-images.githubusercontent.com/49456798/176803861-b056b028-3f7f-4e87-b069-0538b0dbf0b3.png)

**Comment by huabin** [comment]
@BlockArchitech 

Please add more information, thank you.

**Reply by gibbiemonster** [reply]
Instead of "user starred a repository" it says "user published a discussion" even if discussions aren't enabled in that repository.

**Comment by huabin** [comment]
Hello @BlockArchitech 

I tried on iPhone and iPad, and there's no problems exist as you said.
The reason you met this problem might be because you starred a discussion. I guess.
BTW, from your screenshot, it also proves my suspicion.

PS. actually, two days ago I replied to you with the above paragraph. But from your title and content, you're talking about two things that I can't be sure of. So I've edited my reply. But now I think the above paragraph will help.

**Comment by gibbiemonster** [comment]
I can show you tomorrow, but on desktop it says Started the repository. It might just be an android thing.

**Reply by huabin** [reply]
Okay.

**Reply by gibbiemonster** [reply]
This is the desktop screenshot:
<img width="685" alt="Screen Shot 2022-07-03 at 7 34 37 AM" src="https://user-images.githubusercontent.com/49456798/177037714-5b0c2617-6005-4307-af35-cd1856c4b057.png">

Hopefully this clears things up.

**Comment by iskae** [comment, marked as answer]
Hey @BlockArchitech 👋 

Thank you so much for reaching out ❤️ 

It looks like this is a bug in our Android app and I have created an issue for it 👍 We will be fixing it in our next release 🎉 

Cheers!
